ui/src/components/AppLayout.tsx

```diff
@@ -72,7 +72,8 @@ const AppLayout: React.FunctionComponent<IAppLayout> = ({ children }) => {
         { path: '/contribute/knowledge', label: 'Knowledge' },
       ],
     },
-    { path: '/chat', label: 'Chat' },
+    { path: '/granitechat', label: 'Granite-7b Chat' },
```

Comment:
Just a suggestion (feel free to ignore) : Won't it be better to use a single chat window where it  can provide a drop down option to select the model to chat with? Mainly that will probably be more convenient, if in future we want to add more models for chat, so that side bar list wont grow.

Reply:
Yeah agree, I have some changes consolidating the route. It was just a time thing to implement but Ive re-done the layout, will post soon.

Reply:
Done ty.